oap-server/server-bootstrap/src/main/resources/ui-initialized-templates.yml

```
@@ -1067,4 +1067,124 @@ templates:
     # False means providing a basic template, user needs to add it manually.
     activated: true
     # True means wouldn't show up on the dashboard. Only keeps the definition in the storage.
+    disabled: false
+  - name: Spring-Sleuth
+    type: "DASHBOARD"
+    configuration: |-
+      [
+        {
+          "name":"Spring ",
```

```suggestion
          "name":"Spring Sleuth",
```

This seems much easier than `meter-receive-config/spring.yaml` provided, could you share why? I expect they are matched and used by the end-user directly.

---

And notice, this change effects your screenshot, please consider the name. Because you have whitespace after `Spring`, I assume you are missing something.